atd-vze/src/views/Crashes/Crash.js

```diff
@@ -1,19 +1,33 @@
-import React from "react";
-import { Card, CardBody, CardHeader, Col, Row, Table, Alert } from "reactstrap";
+import React, { useState, setState } from "react";
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Col,
+  Row,
+  Table,
+  Alert,
+  Input,
+} from "reactstrap";
 import { withApollo } from "react-apollo";
 import { useQuery } from "@apollo/react-hooks";
-import crashDataMap from "./crashDataMap";
+import { crashDataMap, geoFields } from "./crashDataMap";
```

Comment:
It appears neither setState or geoFields is being used, once removed this should be good to ship.

Reply:
fixed in another branch